rebuttal.tex

```diff
@@ -357,6 +357,21 @@ \section{Reviewer 3}
 \item The current conclusion works fine as the finishing section of the paper.
 \end{enumerate}
 \begin{solution}
+  FIXME.
+  \begin{enumerate}
+    \item I believe we shouldn't move ``Other Projects'' section to the
+      main paper content.  It either should stay in the supplement or we
+      should do a moderate cleanup for this section.  E.g. list only few
+      projects here and put other in the supplement.  Or/And show here
+      only project names.
```

Comment:
I agree with this, but I think we'll need a stronger argument if we are going to leave it out.